packages/cozy-harvest-lib/src/models/KonnectorJob.js

```diff
@@ -34,6 +34,21 @@ export const RUNNING_TWOFA = 'RUNNING_TWOFA'
 export const TWO_FA_MISMATCH = 'TWO_FA_MISMATCH'
 export const TWO_FA_REQUEST = 'TWO_FA_REQUEST'
 
+const JOB_EVENTS = [
+  ERROR_EVENT,
+  LOGIN_SUCCESS_EVENT,
+  SUCCESS_EVENT
+]
+
+const eventToStatus = {
+  [ERROR_EVENT]: ERRORED,
+  [LOGIN_SUCCESS_EVENT]: LOGIN_SUCCESS,
+  [SUCCESS_EVENT]: SUCCESS
+}
+const stepEvents = [LOGIN_SUCCESS_EVENT]
+const isStatusEvent = eventName => Boolean(jobEventToStatus[eventName])
```

Comment:
Shouldn't it be `eventToStatus[eventName]`?
(maybe fixed later, I'm reviewing https://github.com/cozy/cozy-libs/pull/956/commits/3a1af74c27c412f4df11b31327bbee221c074c4d)